uaclient/tests/test_gpg.py

```diff
@@ -0,0 +1,70 @@
+from uaclient import exceptions, gpg, util
+from uaclient.testing import data
+
+import logging
```

Comment:
stdlib imports [should](https://www.python.org/dev/peps/pep-0008/#imports) go above our own imports.